.github/workflows/lint-codeql.yaml

```diff
@@ -2,13 +2,10 @@ name: codeql
 
 on:
   push:
-    branches:
-    - v1.10
-    - v1.9
-    - v1.8
-  pull_request:
     branches:
     - master
+    - v*.*
```

Comment:
I'm not sure if I follow this. This will mean that any branch starting with `v` would be triggered by this, it's not always the case. There are branches starting with `v` that do not belong to stable branches.